### ISSUE TITLE
Add reference "Working Open" from Mozilla

### ISSUE THREAD

**Issue by pdurbin** [good-labs/slopi-communication#22]
Unlike Debian (#4), Apache (#5), and GitLab (#6), I haven't found (or really looked for) anything from Mozilla on their communication style but a new blog post by @ahal at https://ahal.ca/blog/2019/fragmented-communication/ called "The Cost of Fragmented Communication" indicates that Mozilla uses a SLOPI communication style. Here are some quotes from that post:

"Mozilla walks the walk when it comes to involving the community. Much of our communication, decision making and implementation all happens right out in the open. Anyone can go back and read almost everything I’ve ever posted to Bugzilla, every message I left on IRC, every post I’ve made to the newsgroups, every patch I’ve submitted or review I’ve conducted. It’s all out there in the public. For most, this level of openness is a very strange concept. For many it’s unsettling. But it allows Mozilla to bring community into the conversation unlike anywhere else I’ve ever seen at this scale. This is a feature, a huge competitive advantage."

"Now Mozilla had (and still has) two communication platforms. IRC for public community facing discussions, Slack for private, NDA’ed only discussions."

"Most of the conversation I see on Slack has no reason to be private"

"The previous section purposefully doesn’t mention the words ‘private’ or ‘public’. My hope is to convince you that simply giving people a choice (regardless of the mode of communication) causes problems worth solving in and of themselves. But remember that people usually tend to use Slack, not because they are having private conversations, but because they prefer it to IRC. This is a problem because Slack is exclusionary. Contributors can’t access it without signing an NDA (which requires employee intervention). These conversations exclude the wider Mozilla community, treating contributors as second class citizens"

"Exclusive conversations that have no reason to be private, arising (presumably) when a critical mass of people prefer the closed platform to the open one."

"The process to replace IRC should result in a public communication platform that is more or less on par with Slack in terms of bells and whistles. This should incentivize more people over to the side of public by default and reduce the impact of both of the aforementioned problems."

"Ideally the IRC replacement will be able to handle both public and private modes of communication."

These are some great quotes and he even links to a conversation he had in 2010: https://mozilla.logbot.info/ateam/20100325#c4178 . :smile: I love it.

**Comment by pdurbin** [comment]
Today I left the following comment at on the blog post I mentioned above:

"Hi! This is a fantastic blog post and I quoted several points you made about Mozilla’s communication style at https://github.com/good-labs/slopi-communication/issues/22

I’d love to find something under mozilla.org about Mozilla’s communication style, especially with regard to decision making happening in the open and being logged or otherwise record in public places (you mentioned Bugzilla and IRC logs, for example).

Andrew, can you please help me find a reference? Thanks again for this great blog post!"

@ahal commented back right away (thanks!) with this reply:

"Hi, and thanks! I’m not aware of any official stance or documentation with regards to public record keeping. It’s just how it’s always been and many of us old-timers just take it for granted (though as you might tell from this post, in recent years it’s been less like this).

The logbot site I linked is a side project from a Mozilla employee, and I believe the older logs have been saved and passed on from Mozillians who ran logging scripts back in the day. I share your belief that having publicly indexed logs is benecifial. Hopefully we’ll continue to have this ability with whatever replaces IRC."

I'm going to assume this means that this means that Mozilla doesn't have anything official written up about this but maybe I'll see if I can get in touch with other Mozilla people I'm met in person like @acabunoc or have chatter with like @kaythaney . I feel like the best non-Mozilla example I've found so far is Apache's embrace of what they call "Open Communications" as I've quoted in #5.

**Comment by acabunoc** [comment]
Hey! Take a look at the Working Open page on Mozilla’s wiki:
https://wiki.mozilla.org/Working_open

**Comment by pdurbin** [comment]
@acabunoc thank you!! There are some great resources there and by clicking "What links here" I found even more linked from https://wiki.mozilla.org/Nature_of_Mozilla !! :tada: :tada: :tada: Much appreciated!

**Comment by pdurbin** [comment]
In pull request #27 I added a reference to the wiki page above to the README. I tried to watch some of the videos but somewhat ironically they all seemed to require auth. :cry: Also, a lot of the etherpad links don't work anymore.

I didn't add a link to @ahal 's blog post but (again) I love it and I'd be happy to.

I'm sort of wondering if I should reach out to @globau because he runs an IRC logging bot just like I do.

I think I'll leave this issue open for now but I'm glad to at least get a little bit of an official link to Mozilla's thoughts on this topic. Thanks again for unblocking me, @acabunoc !
